dotnet-core/plan.sh

```diff
@@ -25,34 +25,33 @@ pkg_deps=(
 pkg_build_deps=(
   core/patchelf
 )
-pkg_bin_dirs=(bin)
+pkg_bin_dirs=(.)
 
 do_unpack() {
   # Extract into $pkg_dirname instead of straight into $HAB_CACHE_SRC_PATH.
-  mkdir -p "$pkg_dirname"
-  tar xfz "$pkg_filename" -C "$pkg_dirname"
+  mkdir -p "$HAB_CACHE_SRC_PATH/$pkg_dirname"
+  tar xfz "$HAB_CACHE_SRC_PATH/$pkg_filename" -C "$HAB_CACHE_SRC_PATH/$pkg_dirname"
 }
 
 do_prepare() {
-  patchelf --interpreter "$(pkg_path_for glibc)/lib/ld-linux-x86-64.so.2" \
-    --set-rpath "$LD_RUN_PATH" \
-    ./dotnet
+  find -type f -name 'dotnet' \
+    -exec patchelf --interpreter "$(pkg_path_for glibc)/lib/ld-linux-x86-64.so.2" --set-rpath $LD_RUN_PATH {} \;
+  find -type f -name '*.so*' \
+    -exec patchelf --set-rpath $LD_RUN_PATH {} \;
 }
 
 do_build() {
   return 0
 }
 
 do_install() {
-  # Copy the files into the package
   cp -a . "$pkg_prefix"
+}
 
-  # Wrap the binary in a script that sets the library paths
-  mkdir -p "$pkg_prefix/bin"
-  cat > "$pkg_prefix/bin/dotnet" <<EOF
-#!/bin/sh
-export LD_LIBRARY_PATH="$pkg_prefix/lib:$LD_RUN_PATH:\$LD_LIBRARY_PATH"
-exec $pkg_prefix/dotnet \$@
-EOF
-  chmod +x "$pkg_prefix/bin/dotnet"
+do_check() {
+  ./dotnet --version
```

Comment:
Yeah we can do a:
```
dotnet new
dotnet run
```
which creates a hello world console app and runs it. That should be sufficient.